wp-content/themes/core/components/header/subheader/Subheader_Controller.php

```diff
@@ -50,8 +71,71 @@ public function get_title_args(): array {
 
 		return [
 			Text_Controller::TAG     => 'h1',
-			Text_Controller::CLASSES => [ 'page-title', 'h1' ],
+			Text_Controller::CLASSES => [ 'page-title', 'h1', 'c-subheader__title' ],
 			Text_Controller::CONTENT => $this->get_page_title(),
 		];
 	}
+
+		/**
+	 * @return string
+	 */
+	public function get_container_classes(): string {
+		return Markup_Utils::class_attribute( $this->container_classes );
+	}
+
+	/**
+	 * @return string
+	 */
+	public function get_media_classes(): string {
+		return Markup_Utils::class_attribute( $this->media_classes );
+	}
+
+	/**
+	 * @return string
+	 */
+	public function get_content_classes(): string {
+		return Markup_Utils::class_attribute( $this->content_classes );
+	}
+
+	public function get_image_args() {
+		if ( ! has_post_thumbnail() ) {
+			return [];
+		}
+
+		return [
+			Image_Controller::IMG_ID => (int) get_post_thumbnail_id(),
+			Image_Controller::AS_BG        => true,
+			Image_Controller::AUTO_SHIM    => false,
+			Image_Controller::USE_LAZYLOAD => true,
+			Image_Controller::WRAPPER_TAG  => 'div',
+			Image_Controller::CLASSES      => [ 'c-image--bg' ],
+			Image_Controller::IMG_CLASSES  => [ 'c-subheader__media__image' ],
+			Image_Controller::SRC_SIZE     => Image_Sizes::SIXTEEN_NINE,
+			Image_Controller::SRCSET_SIZES => [
+				Image_Sizes::SIXTEEN_NINE,
+				Image_Sizes::SIXTEEN_NINE_SMALL,
+			],
+		];
+	}
+
+	public function render_breadcrumbs(): void {
+		
+		get_template_part(
+			'components/breadcrumbs/breadcrumbs',
+			'page',
```

Comment:
Can we remove this `page` modifier? There's no page-specific breadcrumb template at this time. The correct value here is `null`.